signature/policy_eval.go

```diff
@@ -142,7 +142,7 @@ func (pc *PolicyContext) requirementsForImageRef(ref types.ImageReference) Polic
 			return req
 		}
 
-		// Look for a match of the possible parent namespaces.
+		// Look for a match of the possible parent namespaces and wildcarded subdomains.
```

Comment:
As far as the generic code cares, wildcards are just parent PolicyConfigurationNamespaces.